.azure-pipelines/integration-tests.yml

```diff
@@ -216,22 +211,22 @@ jobs:
 
   steps:
   - task: UseDotNet@2
-    displayName: install dotnet core sdk 2.1
+    displayName: install dotnet core runtime 2.1
     inputs:
-      packageType: sdk
+      packageType: runtime
```

Comment:
Note to reviewers: we only need the newest SDK (3.1.x) to build everything, so install runtimes for 2.1 and 3.0 to maybe save a little time